.goreleaser.yml

```
@@ -0,0 +1,132 @@
+before:
+  hooks:
+    - go version | grep --quiet "go1\.15\.2" || echo "Go binary version must be 1.15.2"
+    - go mod download
+builds:
+  - id: operator-sdk
+    main: ./cmd/operator-sdk
+    binary: operator-sdk
+    env:
+      - CGO_ENABLED=0
+      - GO111MODULE=on
+      - REPO=github.com/operator-framework/operator-sdk
+    mod_timestamp: "{{ .CommitTimestamp }}"
+    asmflags:
+      - all=-trimpath={{ dir .ArtifactPath | dir | dir | dir }}
+    gcflags:
+      - all=-trimpath={{ dir .ArtifactPath | dir | dir | dir }}
+    ldflags:
+      - -X {{ .Env.REPO }}/internal/version.Version={{ if not .IsSnapshot }}v{{ end }}{{ .Version }}
+      - -X {{ .Env.REPO }}/internal/version.GitVersion={{ .Tag }}
+      - -X {{ .Env.REPO }}/internal/version.GitCommit={{ .FullCommit }}
```

Should we just use the same build args we use for the `build` targets?
```suggestion
      - -X {{ .Env.REPO }}/internal/version.Version={{ .Env.SIMPLE_VERSION }}
      - -X {{ .Env.REPO }}/internal/version.GitVersion={{ .Env.GIT_VERSION }}
      - -X {{ .Env.REPO }}/internal/version.GitCommit={{ .Env.GIT_COMMIT }}
```